src/AutoUI/NoRecordsFoundView.tsx

```diff
@@ -37,21 +41,21 @@ export const NoRecordsFoundView = <T extends AutoUIBaseResource<T>>({
 				gap: 3,
 			}}
 		>
-			<Typography variant='titleLg' fontWeight='bold'>
+			<Typography variant="titleLg" fontWeight="bold">
 				{noDataInfo?.title ??
 					t('no_data.no_resource_data_title', {
 						resource: t(`resource.${model.resource}_other`).toLowerCase(),
 					})}
 			</Typography>
 			{noDataInfo?.subtitle && (
-				<Typography variant='title'>{noDataInfo?.subtitle}</Typography>
+				<Typography variant="title">{noDataInfo?.subtitle}</Typography>
 			)}
 			{noDataInfo?.info && (
-				<Alert severity='info' sx={{ my: 3 }}>
+				<Callout severity="info" color="white" sx={{ my: 3 }}>
```

Comment:
Why add the `color="white"`? We should try to avoid this kind of customization as much as possible, was it necessary or just testing?

Reply:
Following our internal discussion, this has been replaced by `variant="subtle"`